src/CoreShop/Component/Variant/README.md

```diff
@@ -0,0 +1,25 @@
+CoreShop Variant Component
+==================
+
+CoreShop
+------
+
+![CoreShop](http://www.coreshop.org/assets/img/coreshop-logo.svg)
+
+CoreShop is an eCommerce Solution for Pimcore. It is build from decoupled components to get highest quality of
+code. [Read more on coreshop.org](http://www.coreshop.org)
+
+Documentation
+-------------
+
+Documentation is available on [**coreshop.org**](https://www.coreshop.org/docs/2.1.0/Components/Variant_Component.html).
```

Comment:
Shouldn't this be:

```suggestion
Documentation is available on [**coreshop.org**](https://docs.coreshop.org/3.0.0/Components/Variant_Component.html).
```

Reply:
yes. should be for all bundles/components, will solve that in a separate PR